This modifies the behaviour of Chromium when used as your default internet
browser to open links in an incognito window.

--- /usr/share/applications/chromium-browser.desktop	2016-04-15 09:35:50.339656952 +1000
+++ /usr/share/applications/chromium-browser.desktop	2016-04-15 09:29:10.279654754 +1000
@@ -167,7 +167,7 @@ Comment[vi]=Truy cập Internet
 Comment[zh_CN]=访问互联网
 Comment[zh_HK]=連線到網際網路
 Comment[zh_TW]=連線到網際網路
-Exec=chromium-browser %U
+Exec=chromium-browser --incognito %U
 Terminal=false
 X-MultipleArgs=false
 Type=Application
